src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/DynamicInterfaceCastableImplementation.cs

```diff
@@ -54,7 +54,7 @@ internal sealed class DynamicInterfaceCastableImplementationAnalyzer : Diagnosti
 
         internal const string NonStaticMemberIsMethodKey = nameof(NonStaticMemberIsMethodKey);
 
-        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(
+        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get; } = ImmutableArray.Create(
```

Comment:
I think that @sharwell rejected these changes at some point saying that the perf gain was somehow null as we know the call is only done once (I might remember wrong though).

Reply:
IIRC it was submitted as part of a PR with something else, but yes, these calls only occur once in practice.

Reply:
Side note: The PR doesn't only update `SupportedDiagnostics`, but also a couple of immutable arrays which may (or may not) be accessed multiple times.